lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -91,6 +93,14 @@ def _get_resource_link_id(program_uuid, request) -> str:
     def _get_result_sourcedid(context_id, resource_link_id, user_id) -> str:
         return f'{context_id}:{resource_link_id}:{user_id}'
 
+    def get_user_roles(self, user):
```

Comment:
why the name states `roles` but the method only returning just one role?

Reply:
The LTI consumer accepts multiple roles as a comma separated string. e.g "instructor,student"
We only send it one role right now

Reply:
Ah I see, thanks for the confirmation. _Resolving comment..._